### PR TITLE
DevWs

### PR DESCRIPTION
**Pull request by computime-jasonchan**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 1]
```diff
@@ -25,14 +25,12 @@ metadata {
 
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B04"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "0003, 000A, 0019", manufacturer: "Jasco Products", model: "45853", deviceJoinName: "GE ZigBee Plug-In Switch", ocfDeviceType: "oic.d.smartplug"
```
Comment (shrakuma) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:28]: @computime-jasonchan  Can you please let us know the reason for the removal of the code here. This fingerprint is for GE and we cannot remove that.

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 2]
```diff
@@ -25,14 +25,12 @@ metadata {
...
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "0003, 000A, 0019", manufacturer: "Jasco Products", model: "45853", deviceJoinName: "GE ZigBee Plug-In Switch"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702, 0B05", outClusters: "000A, 0019", manufacturer: "Jasco Products", model: "45856", deviceJoinName: "GE ZigBee In-Wall Switch"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "PSM_00.00.00.35TC", deviceJoinName: "Ozom Smart Plug", ocfDeviceType: "oic.d.smartplug"
```
Comment (shrakuma) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:30]: @computime-jasonchan  same here -  a piece of code is removed which should not be done as it affects that partner

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 3]
```diff
@@ -25,14 +25,12 @@ metadata {
...
 		fingerprint profileId: "0104", inClusters: "0000, 0702, 0003, 0009, 0B04, 0006, 0004, 0005, 0002", outClusters: "0000, 0019, 000A, 0003, 0406", manufacturer: "Develco Products A/S", model: "Smart16ARelay51AU", deviceJoinName: "Aurora Smart Inline Relay"
 		fingerprint manufacturer: " ", model: "PAN18-v1.0.7", deviceJoinName: "Philio Smart Plug", ocfDeviceType: "oic.d.smartplug" //profileId: "0104", inClusters: "0000, 0003, 0006, 0702", outClusters: "0003, 0019",
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0702", manufacturer: "SALUS",  model: "SX885ZB", deviceJoinName: "Salus miniSmartplug"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B04", outClusters: "0000, 0004", manufacturer: "MEGAMAN", model: "SH-PSUKC44B-E", deviceJoinName: "INGENIUM ZB Smart Power Adaptor", ocfDeviceType: "oic.d.smartplug"
-		fingerprint profileId: "0104", inClusters: "0000, 0702, 0003, 0009, 0B04, 0006, 0004, 0005, 0002", outClusters: "0000, 0019, 000A, 0003, 0406", manufacturer: "Aurora", model: "Smart16ARelay51AU", deviceJoinName: "Aurora Smart Inline Relay"
```
Comment (shrakuma) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: @computime-jasonchan  these 2 lines were removed and these are again production fingerprints for another device. please revert these changes and re-submit a new PR

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy [hunk 4]
```diff
@@ -98,7 +96,7 @@ def refresh() {
 
 def configure() {
 	log.debug "in configure()"
-	if ((device.getDataValue("manufacturer") == "Develco Products A/S") || (device.getDataValue("manufacturer") == "Aurora"))  {
```
Comment (shrakuma) [inline devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:101]: why was the (device.getDataValue("manufacturer") == "Aurora")) removed?

**Comment by shrakuma** [comment]
@computime-jasonchan please take a look at the above comments as they would need to be fixed and possibly may need a new PR.
cc @tpmanley @greens @MAblewiczS

**Review by shrakuma — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 1 above)
Comment (computime-jasonchan) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:28]: The reason is those GE and Aurora code is added after I commit my change.

I have check the master branch already include my change and those GE and Aurora change, Do I still needed to commit another PR?

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 3 above)
Comment (computime-jasonchan) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: The reason is those line is added after my submission.

Master branch already include my change and the change you mentioned, still need another PR?

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 4 above)
Comment (computime-jasonchan) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:101]: The reason is those line is added after my submission.

Master branch already include my change and the change you mentioned, still need another PR?

File: devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy (hunk 3 above)
Comment (greens) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: your PR just needs to be rebased onto the updated `master` branch

Comment (computime-jasonchan) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: I rebased and update again. Please check if that is OK?

Comment (greens) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: No. If you've set the upstream repo of your branch correctly, you should be able to do:

`git fetch upstream master`
`git rebase upstream/master`
`git push origin`

I'm guessing that the `master` branch you're rebasing onto is out of date.

Comment (shrakuma) [reply devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy:35]: @computime-jasonchan can you please take a look at the above comment

**Comment by shrakuma** [comment]
@computime-jasonchan  not sure if my comments came throuhg correctly 
per @greens  please take a look at 
If you've set the upstream repo of your branch correctly, you should be able to do:

git fetch upstream master
git rebase upstream/master
git push origin

I'm guessing that the master branch you're rebasing onto is out of date.

**Comment by tpmanley** [comment]
@computime-jasonchan what fingerprint were you trying to add in this PR? It doesn't look like any new fingerprints are added in this PR (perhaps added in a different one?) so I'm wondering if we can just close this.

**Comment by computime-jasonchan** [comment]
My model is SX885ZB, I have checked that is in Master already.

**Comment by tpmanley** [comment]
Since the fingerprint was already added I'm going to close this PR.

**Pull request closed without merging**